ci_support/run_docker_build.sh

```diff
@@ -37,35 +39,11 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
+conda update --yes --all
+conda install --yes conda-build==1.18.2
 conda info
 
-
-# Embarking on 3 case(s).
-
-    set -x
-    export CONDA_PY=27
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    
-    /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
-    
-    
-
-    set -x
-    export CONDA_PY=34
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    
-    /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
-    
-    
-
-    set -x
-    export CONDA_PY=35
-    set +x
+# Embarking on 1 case(s).
```

Comment:
That looks significantly different. Have these cases been enumerated manually @conda-forge/mingwpy?